framework/include/utils/MooseADWrapper.h

```diff
@@ -25,16 +25,24 @@ struct MooseADWrapperStruct<Real, true>
   typedef ADReal type;
 };
 
+template <>
+struct MooseADWrapperStruct<Point, true>
+{
+  typedef ADPoint type;
+};
+
+// W<T> -> W<ADT>, e.g. std::vector<Real> -> std::vector<ADReal>
 template <template <typename> class W, typename T, bool is_ad>
```

Comment:
Yea I don't think this would match `std::vector` because `std::vector` is a `template <typename, typename>`